combat-trainer.lic

```diff
@@ -3136,6 +3132,9 @@ class GameState
     @use_barb_combos = settings.use_barb_combos
     echo("  @use_barb_combos: #{@use_barb_combos}") if $debug_mode_ct
 
+    @use_analyze_combos = settings.use_analyze_combos
```

Comment:
💡 You could (maybe) simplify these changes with `@use_analyze_combos = settings.use_analyze_combos || @use_barb_combos`

Then you only have one thing to remove later once `@use_barb_combos` can be removed. And move the "# @use_barb_combos needs to be removed given people time to switch" comment up here.

Reply:
Yup good call.